USERGUIDE.md

```diff
@@ -273,88 +273,88 @@ The i386 version of Homebrew can be installed in parallel with the arm64 version
 
-- **Build Configuration vs Project Configuration**: An MSDK project is primarily made up of two complementary systems:  The _Build System_ and the _Integrated Development Environment (IDE)_.  These systems each offer their own configuration interfaces, and it's important to note what each is used for.
+- **Build Configuration vs. Project Configuration**: An MSDK project is comprised of two complementary systems:  The _Build System_ and the _Integrated Development Environment (IDE)_. These systems each offer their own configuration interfaces, and it's important to note what each is used for.
 
-    The **Build System** manages the compilation of source code into program binaries and offers a **Command-Line Interface (CLI)** for setting **Build Configuration Variables**.
+    The **Build System** manages source code compilation into program binaries and offers a **Command-Line Interface (CLI)** for setting **Build Configuration Variables**.
 
-    The **IDE** offers a higher level user interface (typically with a GUI) for managing a project and sits _on top_ of the build system's _CLI_.  Each IDE offers its own settings for managing fundamental aspects of the build such as:
+    The **IDE** offers a higher-level user interface (typically with a GUI) for managing a project and sits _on top_ of the build system's _CLI_. Each IDE offers its own settings for managing fundamental aspects of the build, such as:
 
     - Setting the _Target Microcontroller_
     - Setting the _Board Support Package_
     - Configuring the _Environment_ and _System Path_ for use with the MSDK toolchain
 
 ### Getting Started with Visual Studio Code
 
-The MSDK includes Visual Studio Code (“VS Code”) support via the [VSCode-Maxim](https://github.com/MaximIntegratedTechSupport/VSCode-Maxim) project.
+The MSDK includes Visual Studio Code ("VS Code") support through the [VSCode-Maxim](https://github.com/MaximIntegratedTechSupport/VSCode-Maxim) project.
 
-This section walks through setup, opening, and running an example project with VS Code.  This material is also available in video form targeting the MAX78000 in ["Understanding Artificial Intelligence Episode 8.5 - Visual Studio Code"](https://www.analog.com/en/education/education-library/videos/6313212752112.html).  For full documentation, see the [Visual Studio Code](#visual-studio-code) section of this User Guide.
+This section walks through setup, opening, and running an example project with VS Code. This material is also available in video form targeting the MAX78000 in ["Understanding Artificial Intelligence Episode 8.5 - Visual Studio Code"](https://www.analog.com/en/education/education-library/videos/6313212752112.html). For complete documentation, see the [Visual Studio Code](#visual-studio-code) section of this User Guide.
 
 #### Setup (VS Code)
 
-The setup below only needs to be done once per MSDK [installation](#installation).
+The setup below must only be done once per MSDK [installation](#installation).
```

Comment:
This could imply that the setup can't be redone. Consider changing it back to the original phrasing.